docs/source/tutorials_and_examples/python/virtualenv.md

```diff
@@ -0,0 +1,31 @@
+```python
+class Documentation:
+    RATING = "Acceptable" # Better structure
+```
+
+#Virtualenv
+ 
+- `python -m virtualenv <name>` - initialize a new virtualenv `<name>` in a new folder `<name>` in the current location. Called `evenv` in these docs.
+- `python -m virtualenv -p path/to/alternate/python_executable <name>` - create a virtualenv using another Python version than default.
```

Comment:
May be worth to offer an example as  `-p /usr/bin/python3.7` here. Overall, it can be a good idea to have the user check which is their default version is with `python --version` first.